src/releng_docs/deploy/regular.rst

```diff
@@ -44,6 +86,8 @@ Protocal that we follow is:
    - announce in ``#ci`` channel that a push to production is about to
      happen.
 
+     TODO: direct this to the person on duty
+
      Example message::
 
          I am about to release a new version of mozilla/release-services
```

Comment:
Nit: s/I am/I'm/

Reply:
Single line for the IRC message, otherwise it's a PITA to copy/paste